features/step_definitions/elements_steps.rb

```diff
@@ -26,7 +26,7 @@
   @test_site.home.wait_for_lots_of_links(0.1, count: 2)
   Capybara.using_wait_time(0.3) do
     # intentionally wait and pass nil to force this to cycle
-    expect(@test_site.home.wait_for_lots_of_links(nil, count: 198_108_14)).to be false
+    expect(@test_site.home.wait_for_lots_of_links(nil, count: 198_108_14)).to be_falsey
```

Comment:
Is there a reason this was changed?

Reply:
`be false` will check for the result to be false ONLY, `be_falsey` check for `nil` or `false`